packages/deploy-tooling/src/cli/config.ts

```diff
@@ -192,3 +194,69 @@ export async function mergeConfig(taskConfig: AbapDeployConfig, options: CliOpti
 
     return config;
 }
+
+/**
+ * Display application properties during confirmation prompt.
+ *
+ * @param config - config to be displayed
+ * @param isDeployCmd - show different fields depending on task being executed
+ */
+export function showAppInfo(config: AbapDeployConfig, isDeployCmd = true): void {
+    const displayList = [];
+    const deployStr = isDeployCmd ? 'deploy' : 'undeploy';
+
+    const _getDisplayValue = (field: boolean | string | undefined, defaultValue?: boolean | string) =>
+        field ?? defaultValue ?? '';
+
+    // Ensure this is the first item in the list
+    if (config.lrep || config.adaptation?.namespace) {
+        displayList.unshift(
+            `${chalk.blue('Repository Entry')}: ${_getDisplayValue(
+                config.lrep ?? config.adaptation?.namespace?.toString() ?? ''
+            )}`
+        );
+    } else {
+        displayList.unshift(`${chalk.blue('Application Name')}: ${_getDisplayValue(config.app.name)}`);
+    }
+
+    if (isAppStudio()) {
+        displayList.push(`${chalk.blue('Destination')}: ${_getDisplayValue(config.target.destination)}`);
+    } else {
+        displayList.push(`${chalk.blue('Target')}: ${_getDisplayValue(config.target.url)}`);
+    }
+    if (config.target.service) {
+        displayList.push(`${chalk.blue('SAPUI5 OData Service Path')}: ${_getDisplayValue(config.target.service)}`);
+    }
+
+    console.log();
+    console.log(
+        chalk.blue.bold.underline(
+            `${
+                config.test
+                    ? 'Confirmation is required to ' + deployStr + ' the app in test mode:'
+                    : 'Confirmation is required to ' + deployStr + ' the app:'
+            }`
+        )
+    );
+
+    if (!config.strictSsl) {
+        console.log(
+            chalk.yellow(
+                `You chose not to validate SSL certificate. Please verify the server certificate is trustful before proceeding, refer to https://help.sap.com/viewer/17d50220bcd848aa854c9c182d65b699/Latest/en-US/4b318bede7eb4021a8be385c46c74045.html.`
+            )
+        );
+    }
+
+    displayList.push(`${chalk.blue('Transport Request')}: ${_getDisplayValue(config.app.transport)}`);
+
+    if (isDeployCmd) {
+        displayList.push(`${chalk.blue('Package')}: ${_getDisplayValue(config.app.package)}`);
+        displayList.push(`${chalk.blue('Client')}: ${_getDisplayValue(config.target.client)}`);
+        displayList.push(`${chalk.blue('Cloud')}: ${_getDisplayValue(config.target.scp, false)}`);
+    }
+
+    console.log('');
+    // console.log spacing is important to ensure tabbing of each line when displayed
+    displayList.filter((ele) => ele !== undefined).forEach((ele: string) => console.log(`    ${ele}`));
```

Comment:
I used to write the following for logging indentation. I find it easier to read and maintain different number of whitespaces. `' '.repeat(0)` gives 0 whitespace so it becomes a generic solution for 0-* spaces. 

```
console.log(`${' '.repeat(4)}${ele}`)
```